### ISSUE TITLE
No window when running example on Wayland, Sway

### ISSUE THREAD

**Issue by e00E** [rust-windowing/winit#776]
Maybe duplicate of https://github.com/tomaka/winit/issues/640 .

I was investigating https://github.com/jwilm/alacritty/issues/2041 and tried to run an example:
```
cargo run --example window
    Finished dev [unoptimized + debuginfo] target(s) in 0.06s
     Running `target/debug/examples/window`
WindowEvent { window_id: WindowId(Wayland(WindowId(94592017375568))), event: Refresh }
```

At this point I do not see a window, there is no further output and the command is still running.

```
sway --version
sway version 1.0-beta.2-274-g5f45a4bb (Jan 25 2019, branch 'makepkg')
```

**Comment by vberger** [comment]
As said in the other issue, given how wayland works a window does not exist until its contents have been drawn once.

However the winit examples do not draw anything, hence the windows never appear.

You may prefer to use the glutin examples to work with, as they draw something.

**Comment by vberger** [comment]
I agree it is probably worth it to add some wayland-specific code that blanks the window in the examples, so that they can actually work.

**Comment by ZeGentzy** [comment]
@vberger Has this been resolved with the changes in eventsloop v2?

**Comment by aleksander** [comment]
@ZeGentzy yes, this bug is fixed in eventloop-2.0 branch.

**Comment by vberger** [comment]
Actually, no : https://github.com/rust-windowing/winit/pull/981
